src/axiosConfig.ts

```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
+const AUTH_TOKEN = window.localStorage.getItem("accessToken");
+const instance = axios.create({
+  baseURL: process.env.REACT_APP_LOGIN_API,
+});
+
+if (AUTH_TOKEN)
+  instance.defaults.headers.common["Authorization"] = "Bearer " + AUTH_TOKEN;
```

Comment:
Avoid using shortcuts, try to use proper code structure.

Reply:
are you referring to remove - if (AUTH_TOKEN)